### PR TITLE
Added pgagent RA

### PR DESCRIPTION
**Pull request by OlegPS**
Thanks!

### CODE REVIEW

File: heartbeat/pgagent [hunk 1]
```diff
@@ -0,0 +1,121 @@
+#!/bin/sh
+#
+#       High-Availability nginx OCF resource agent
```
Comment (krig) [inline heartbeat/pgagent]: This description seems wrong, this is not related to nginx, right?

**Comment by OlegPS** [comment]
Ups... My bad, fixed! Kristoffer, thank you!

**Comment by krig** [comment]
There are no other agents that rely on `start-stop-daemon`, and it is not available on all systems (for example, on my openSUSE system, I don't have `start-stop-daemon`).

That might be OK since this is the first version of the agent, but you should probably say in the description which distributions it is written for.

**Comment by oalbrigt** [comment]
I think we should start the daemon directly instead, as that will work on all distributions, and only need changes to any path parameters that might differ from one distribution to the other.

**Comment by OlegPS** [comment]
Reworked now, thanks for usefull comments!

**Comment by oalbrigt** [comment]
Thanks.
